content/en/docs/tasks/tools/_index.md

```diff
@@ -24,8 +29,12 @@ computer (including Windows, macOS and Linux PCs) so that you can try out Kubern
 or for daily development work.
 
 You can follow the official [Get Started!](https://minikube.sigs.k8s.io/docs/start/)
-guide, or read [Install Minikube](/docs/tasks/tools/install-minikube/) if your focus
-is on getting the tool installed.
+guide if your focus is on getting the tool installed.
+
+<form>
+  <button type="submit" formaction="https://minikube.sigs.k8s.io/docs/start/" class="btn btn-primary" aria-label="View Minikube Get Started! Guide">View Minikube Get Started! Guide</button>
```

Comment:
nit: minikube is typically written in lowercase, like `kubectl` and `kind`.
Are you sticking with the form element? You could always log an issue to address future work
on the buttons (and localization).

Reply:
+1 to this nit

Reply:
@kbhawkey what I think would be most useful in this case is creating an issue to make a `{{< button text="blah" target="blah" >}}` shortcode.

Reply:
> nit: minikube is typically written in lowercase, like `kubectl` and `kind`.

Agreed, this I'll update

> Are you sticking with the form element?

I think I have to - I don't want to introduce JavaScript, and I'm not sure how to make `<button>` (or `<input>`) work as a link in HTML without the `<form>` wrap. That said, I don't do a lot of front end development, so I may be missing a technique and would be happy to be shown a better way to do this. I think the alternative right now if we don't want the `<form>` element is to just remove the buttons entirely, and I think there is value to having them. 

> You could always log an issue to address future work on the buttons (and localization).

+1.

Reply:
> That said, I don't do a lot of front end development, so I may be missing a technique and would be happy to be shown a better way to do this.

After a chat with @celestehorgan, looks like I could use Bootstrap's buttons https://getbootstrap.com/docs/4.5/components/buttons/#button-tags

I assume this would be preferable to the HTML so I'll update.

Reply:
My latest push updates for the Bootstrap buttons, as well as `minikube` capitalization throughout the page.